pkg/invoke/args.go

```diff
@@ -29,14 +32,18 @@ type inherited struct{}
 
 var inheritArgsFromEnv inherited
 
-func (_ *inherited) AsEnv() []string {
+func (i *inherited) AsEnv() []string {
+	// It stands in for "just use the calling process's environment"
+	i = i
```

Comment:
The warning of the initial code is "pkg/invoke/args.go|35 col 1| receiver name should not be an underscore".

So, I changed the code to 
```
func (i *inherited) AsEnv() []string {
    // It stands in for "just use the calling process's environment"
    i = i // Because i is not used, so I added it.
    
    return nil
}
```
In fact, it is okay if a input parameter is not used in golang, so I will removed "i = i".